test/e2e/framework/network/utils.go

```diff
@@ -83,8 +87,11 @@ const (
 var NetexecImageName = imageutils.GetE2EImage(imageutils.Agnhost)
 
 // NewNetworkingTestConfig creates and sets up a new test config helper.
-func NewNetworkingTestConfig(f *framework.Framework, hostNetwork bool) *NetworkingTestConfig {
+func NewNetworkingTestConfig(f *framework.Framework, hostNetwork bool, options ...func(*NetworkingTestConfig)) *NetworkingTestConfig {
 	config := &NetworkingTestConfig{f: f, Namespace: f.Namespace.Name, HostNetwork: hostNetwork}
+	for _, o := range options {
+		o(config)
```

Comment:
not sure about this, stylistically, although I guess it's kind of necessary given that this function both creates and then uses the config object...

Reply:
It's a pattern I found pretty useful to avoid bloating functions signatures. 
Dave Cheney gave a full talk about it: https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis
But if you prefer, I can just add a `SCTPEnabled` extra parameter.

Reply:
let's pass `SCTPEnabled` for now. the function having 3 args is not a concern yet.

Reply:
Done